app/services/api/v1/ecf/participants_query.rb

```diff
@@ -4,26 +4,16 @@ module Api
   module V1
     module ECF
       class ParticipantsQuery
+        include Api::V3::Concerns::FilterCohorts
```

Comment:
could we keep v3 concerns out of v1 to avoid unintentionally changing v1 when we amend v3 thinking it's only used there? unless we move the concerns out of versioning, let me know your thoughts